package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.5.4",
   "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
   "main": "snabbdom.js",
-  "typings": "type-definitions/snabbdom.d.ts",
+  "typings": "snabbdom.d.ts",
```

Comment:
Is this needed? Since `declaration` is `true` in `tsconfig.json` matching `*.d.ts` should be created for every compiled file and TypeScript should pick those up automatically?